specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/stable/2024-04-03/examples/MsixImage_Expand_Post.json

```diff
@@ -0,0 +1,99 @@
+{
+  "parameters": {
+    "subscriptionId": "daefabc0-95b4-48b3-b645-8a753a63c4fa",
+    "resourceGroupName": "resourceGroup1",
+    "hostPoolName": "hostpool1",
+    "api-version": "2024-04-03",
+    "msixImageURI": {
+      "uri": "imagepath"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "hostpool1/expandmsiximage",
+            "id": "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourcegroups/resourceGroup1/providers/Microsoft.DesktopVirtualization/hostPools/hostpool1/expandMsixImage",
```

Comment:
Can I get an explanation on why this ID is failing Swagger model validation? I am getting an Arm ID Format failure, but I don't know what the issue is.

Reply:
> Can I get an explanation on why this ID is failing Swagger model validation? I am getting an Arm ID Format failure, but I don't know what the issue is.

@ramoka178 , can you answer this? We would like to get this PR finished up so we can use it as a base for a public preview.

Reply:
I'm doubting if its because 

Microsoft.DesktopVirtualization/hostPools/hostpool1/expandMsixImage at the end. 

as it has "hostpool1/expandMsixImage" and causing an extra / in the resource name.

Reply:
Could you try if making a change there works ? or if this the actual format you support and if the tool needs to be updated ? Please put the PR back in queue after this.

Reply:
that is the actual format, this is an operation that operates on a specific hostpool (hostpool1).

Reply:
This code has also already been deployed to our previous stable API, with no trouble.